### DISCUSSION TITLE
Problems compiling

### DISCUSSION THREAD

**Discussion by Lectorn12** [sannybuilder/dev#177] [Q&A]
Hi, so... I recently resintalled my windows  and now I cannont compile .cs files anymore...
I get some sort of error about unknown opcodes...
[Screenshot](https://imgur.com/a/IrIuQe8) - here is the error

Before reinstalling the windows, I just downloaded Sanny Builder 3 and opened a .cs file, edited and compiled without any problems, now can't edit the same files...

**Comment by x87** [comment]
add directive `{$USE ini}`

read more here https://docs.sannybuilder.com/edit-modes/extensions
